packages/widgetbook_cli/bin/command/analyze.dart

```diff
@@ -0,0 +1,201 @@
+import 'dart:io';
+
+import 'package:interact/interact.dart';
+import 'package:mason_logger/mason_logger.dart';
+import 'package:pubspec/pubspec.dart';
```

Comment:
Good that you've looked for a package. How maintained is this?

Reply:
Yes seems like , last review was
Published [3 months ago](https://pub.dev/packages/pubspec)

Reply:
Can you check open issues? But sounds fine

Reply:
Yes I did check , there is only 2 issues  at the ,moment , nothing major that can affect  the usage and implementation in the cli